spring-webclient-mocking/src/test/java/com/baeldung/webclientmocking/service/EmployeeServiceMockWebServerTest.java

```diff
@@ -0,0 +1,124 @@
+package com.baeldung.webclientmocking.service;
+
+import com.baeldung.webclientmocking.domain.Employee;
+import com.baeldung.webclientmocking.enums.Role;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.Rule;
+import org.junit.jupiter.api.*;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EmployeeServiceMockWebServerTest {
```

Comment:
This is a `JUnit 5` test class, so is not `public` by convention - just leave the access modifier off to make it package private.